packages/components/calendar/src/range-date-picker.tsx

```diff
@@ -157,6 +159,10 @@ export const RangeDatePicker = forwardRef<RangeDatePickerProps, "input">(
 
             <Portal {...portalProps}>
               <PopoverContent
+                id={id}
```

Comment:
I was wondering if it would be better to include these areas in the `getCalendarProps` of the `useRangeDatePicker` if possible.

Reply:
I've addressed this in another issue, but it seems to need to be granted here

https://github.com/yamada-ui/yamada-ui/pull/2204

Reply:
Thanks for reference.
If i think about it, this is a popover props, not a calendar, so it shouldn't be in getCalendarProps.
Sorry.